extensions/wrapper/wrapper-ee-api/src/main/java/de/sovity/edc/ext/wrapper/api/ee/model/FileStorage.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ee.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+import java.math.BigInteger;
+import java.util.Date;
+import java.util.Map;
+
+@Data
+@Schema(description = "Available and used resources of a connector.")
+public class FileStorage {
+    @NotBlank
```

Comment:
So as agreed I will remove any annotation from the POJO members since it will not be reflected to the swagger docs. So the actual implementation will deal with it.